tests/resources/example_virtualenv_project/requirements.txt

```
@@ -1,2 +1,2 @@
-mlflow
+/home/akira/akira/oss/mlflow
```

```suggestion
mlflow
```

Can we revert this change?